troubadix/plugins/script_calls_recommended.py

```diff
@@ -79,7 +79,7 @@ def check_content(
             )
         for call in recommended_single_call:
             if not _get_special_script_tag_pattern(
-                name=call, value=".*"
+                name=call, value="(?s:.)*"
```

Comment:
```suggestion
                name=call, value=".*", flags=re.DOTALL
```
Is there any disadvantage in using `re.DOTALL` flag instead of this? If not, I'd prefer this, because we don't have the constrains the author on Stackoverflow has and it's easier to ready IMHO :)

Reply:
I found it nicer with the local group because I'd does make it "independent" of the outer, non-visible pattern.

Currently there is no downside to use `flags=re.DOTALL`